sound/mpd/Makefile

```diff
@@ -101,104 +98,90 @@ define Package/mpd-avahi-service/conffiles
 /etc/avahi/services/mpd.service
 endef
 
-TARGET_CFLAGS += -ggdb3
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib $(if $(ICONV_FULL),-liconv)
-EXTRA_CXXFLAGS += $(if $(CONFIG_GCC_VERSION_4_8),-std=gnu++11,-std=gnu++14)
+EXTRA_CXXFLAGS += -std=gnu++14
 
 CONFIGURE_ARGS += \
 	$(call autoconf_bool,CONFIG_IPV6,ipv6) \
 	--disable-debug \
 	--disable-documentation \
 	--disable-test \
-	--disable-werror \
-	\
+	--disable-aac \
+	--disable-adplugin \
 	--disable-ao \
+	--disable-audiofile \
 	--disable-bzip2 \
+	--disable-cdio-paranoia \
 	--disable-fluidsynth \
 	--disable-wildmidi \
 	--disable-gme \
 	--enable-inotify \
 	--disable-icu \
-	--disable-eventfd \
 	--disable-iso9660 \
 	--disable-jack \
 	--disable-roar \
 	--disable-libwrap \
 	--disable-lsr \
+	--disable-mad \
 	--disable-mikmod \
 	--disable-modplug \
 	--disable-mpc \
-	--disable-mpg123 \
+	--disable-nfs \
 	--disable-openal \
 	--disable-opus \
 	--disable-pulse \
 	--disable-sidplay \
+	--disable-smbclient \
+	--disable-sndfile \
 	--disable-solaris-output \
 	--disable-sqlite \
+	--disable-systemd-daemon \
 	--disable-lame-encoder \
 	--disable-twolame-encoder \
 	--disable-shine-encoder \
+	--disable-vorbis-encoder \
 	--enable-wave-encoder \
 	--disable-wavpack \
+	--disable-webdav \
 	--disable-wildmidi \
 	--disable-zzip \
 	--with-zeroconf=no \
 	--disable-soxr \
-	\
 	--enable-curl \
-	--enable-flac \
```

Comment:
Well, your call.  I prefer to enumerate as much as possible so that it's self-documenting, even to someone not familiar with the defaults.